openqa-trigger-bisect-jobs

```diff
@@ -183,6 +185,23 @@ def find_changed_issues(investigation):
 
 def main(args):
     parsed_url = urlparse(args.url)
+    result_url = urlparse(
+        urlunparse(
+            (
+                parsed_url.scheme,
+                parsed_url.netloc,
+                '/api/v1/jobs/' + parsed_url.path.rsplit('/', 1)[1],
+                "",
+                "",
+                "",
+            )
+        )
+    )
+    log.debug("Retrieving result info from %s" % result_url.geturl())
+    result = fetch_url(result_url, request_type="json")
```

Comment:
The job data is already fetched, see line 208 in the current version.
So I suggest to not do that twice and just move the code up.